modules/microshift-restoring-backups.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * microshift/microshift-auto-recover-manual-backup.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-restoring-backups_{context}"]
+= Restoring backups
+
+Backups can be restored after updates or after other system events that remove or damage required data.
```

Comment:
I'm not sure about "restoring after updates".
If MicroShift is updated to newer version, technically there's no backup matching the version of the data yet.

Reply:
removed